views/content/token.md

```diff
@@ -90,10 +90,11 @@ If you just want to copy paste the code, then use this:
         }
 
         function burnFrom(address _from, uint256 _value) returns (bool success) {
-            require(balanceOf[_from] >= _value);                // Check if the sender has enough
+            require(balanceOf[_from] >= _value);                // Check if the targeted balance has enough
```

Comment:
Maybe change the comment to:
Check if the targeted balance **is** enough

Sounds better imho.